common.webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+
+    context: path.join(__dirname, 'src'),
+
+    entry: {
+        app: './index.jsx'
+    },
+
+    output: {
+        path: __dirname + '/dist',
+        filename: 'main.js'
+    },
+
+    resolve: {
+        extensions: ['.js', '.jsx']
```

Comment:
And could you move this `resolve: {extensions: ['js', 'jsx']}` to `rules` for babel-loader?

Reply:
Done